Userland/Libraries/LibELF/ELFABI.h

```diff
@@ -204,6 +204,7 @@ typedef struct {
 #define EM_X86_64 EM_AMD64
 #define EM_VAX 75      /* DEC VAX */
 #define EM_AARCH64 183 /* ARM 64-bit architecture (AArch64) */
+#define EM_RISCV 243   /* RISC-V */
```

Comment:
Note: I don’t believe this is necessary to make the userland compile, it just prevents our LibELF to read any binaries. As discussed on Discord I have a bare-minimum patchset to allow LibELF to read RISC-V ELF files, which includes this among other things. However, we can leave this in.